### PR TITLE
[RUM] update custom actions guide

### PR DESCRIPTION
**Pull request by hdelaby**
### What does this PR do?

Update RUM custom actions guide

### Motivation

New attributes names and interface

### Preview

https://docs-staging.datadoghq.com/hdelaby/update-custom-actions/real_user_monitoring/guide/send-custom-user-actions/

### Additional Notes

---

### Reviewer checklist
- [ ] Review the changed files.
- [ ] Review the URLs listed in the [Preview](#preview) section.
- [ ] Review any mentions of "Contact Datadog support" for internal support documentation.

### CODE REVIEW

File: content/en/real_user_monitoring/guide/send-custom-user-actions.md [hunk 1]
```diff
@@ -9,43 +9,44 @@ further_reading:
 
 ## Overview
 
-Datadog’s RUM can collect interesting events and timings with Custom User Actions. As an example throughout this guide, we collect user checkouts information from an e-commerce website.
+Real User Monitoring [automatically collects actions][1] on your web application. You may also want to collect additional events and timings such as form completions and business transactions. With custom RUM actions, monitor any interesting event with all the relevant context attached. As an example throughout this guide, we collect user checkouts information from an e-commerce website.
```
Comment (bcaudan) [inline content/en/real_user_monitoring/guide/send-custom-user-actions.md:12]: What about also updating  the title? user action -> action

File: content/en/real_user_monitoring/guide/send-custom-user-actions.md [hunk 2]
```diff
@@ -9,43 +9,44 @@ further_reading:
...
 
 ## 1. Instrument your code
-The `addUserAction` API lets you attach as many attributes as you want in the form of a JavaScript object. In the example, information about the cart is sent when the user clicks the checkout button.
+To create a new RUM action, use the `addUserAction` API. Give your action a name and then attach context attributes in the form of a JavaScript object. In the following example, a `checkout` action is created with details about the user cart when the user clicks on the checkout button.
 
-```
+```javascript
 function onCheckoutButtonClick(cart) {
     DD_RUM.addUserAction('checkout', {
```
Comment (bcaudan) [inline content/en/real_user_monitoring/guide/send-custom-user-actions.md:19]: We now provide a `addAction` API, it could be nicer to document this one

**Comment by hdelaby** [comment]
Hey @apigirl we'd like to rename that page to "Send RUM Custom Actions" (/guide/send-rum-custom-actions) and make an alias from the existing URL to the new one. How should I proceed?

**Review by bcaudan — APPROVED** [review]
LGTM

**Comment by apigirl** [comment]
@hdelaby, I just renamed and added a redirect for you

**Review by apigirl — APPROVED** [review]

**Comment by apigirl** [comment]
otherwise this lgtm!

**Pull request merged**